src/constraint_system/composer.rs

```diff
@@ -76,6 +79,8 @@ pub struct StandardComposer {
     // N.B. They should not be exposed to the end user once added into the composer
     pub(crate) variables: HashMap<Variable, Scalar>,
 
+    pub(crate) jubjub_variables: HashMap<Variable, Fr>,
+
```

Comment:
I don't think that this is the way of handling JubJub Scalars here. 

You commit them as WNAF terms, not as JubJubScalar. @kevaundray , any inputs on that?